### PR TITLE
More consistency and improvements

### PR DESCRIPTION
**Pull request by MikeJeffers**

### CODE REVIEW

File: egress/README.md [hunk 1]
```diff
@@ -99,59 +99,47 @@ application using SAM!
...
-by running:
+Before we can deploy anything, we need an S3 bucket where we can upload our
+Lambda function packaged as a ZIP file. If you don't have an S3 bucket to store
+code artifacts, then this is a good time to create one:
 
-```
-sam local start-api --port 5000 --env-vars env.json --host 0.0.0.0
+```bash
+BUCKET=BUCKET_NAME
+aws s3 mb s3://${BUCKET}
```
Comment (brandonparis) [inline egress/README.md:113]: You need to make sure they have the aws cli, or is that done elsewhere?

**Review by brandonparis — COMMENTED** [review]
aws cli installation link?

File: egress/README.md (hunk 1 above)
Comment (MikeJeffers) [reply egress/README.md:113]: Further up in this readme, not in the diff

**Review by brandonparis — APPROVED** [review]

File: egress/src/app.py [hunk 2]
```diff
@@ -85,7 +124,7 @@ def lambda_handler(event, context):
         job_output_path = os.path.join("/tmp", job_id + ".json")
 
         # Create an authenticated connection to the API with your token
-        api = API(token=Token(TOKEN_DICT))
+        api = API(token=Token.from_str(get_secret()))
```
Comment (brimoor) [inline egress/src/app.py:127]: so beautiful

**Pull request merged**